runtime/include/chpl-gpu.h

```diff
@@ -201,6 +201,15 @@ GPU_CUB_WRAP(DECL_ONE_SORT, keys);
 
 #undef DECL_ONE_SORT
 
+#else // HAS_GPU_LOCALE
+
+// Provide a fallback for the chpl_assert_on_gpu function for non-GPU locales.
+// This works exactly the same as the standard one.
+
+static inline void chpl_assert_on_gpu(int32_t lineno, int32_t filenameIdx) {
```

Comment:
Consider `ln` and `fn` for formal names to have consistency in the runtime. (I don't love those names at all, I love consistency)